src/adapters/MeetingsJSONAdapter.js

```diff
@@ -384,6 +394,27 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     return captureStream;
   }
 
+  /**
+   * Returns available media devices.
+   *
+   * @returns {Array} Array containing media devices
+   * @private
+   */
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
- you can move it out of the class or make it static if it does not need "this"
- getAvailableDevices seems more suggestive
- jsdoc the `type` param

Reply:
ok